src/mem/AbstractMemory.py

```diff
@@ -76,3 +76,5 @@ class AbstractMemory(ClockedObject):
     )
 
     writeable = Param.Bool(True, "Allow writes to this memory")
+
+    collect_stats = Param.Bool(True, "Collect traffic statistics")
```

Comment:
```suggestion
    collect_stats = Param.Bool(True, "Collect statistics per requestor for "
                               "each type of access. Set this to `False` if "
                               "requestors may be unknown or when running "
                               "with multiple `System` objects without a "
                               "`SysBridge`.")
```

Reply:
Thanks for the suggestion. I updated the commit. Please check :)